pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs

```diff
@@ -152,12 +152,12 @@
         Submit your answer to each question with the <strong>Grade</strong> button on the question page.
         Look at <strong>Best submission</strong> to confirm that each question has been graded.
 
-        <% if (authz_result.mode == 'SEB') { %>
+        <% if (authz_result.mode == 'SEB' || 'password' in authz_result) { %>
             <form method="POST" class="form-inline">
             <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
             <input type="hidden" name="__action" value="finish">
             When you are done:
-            <button type="submit" class="btn btn-success">Click here to finish your exam and close the Safe Exam Browser.</button>
+            <button type="submit" class="btn btn-danger" onClick="return confirm('Finishing the exam means you can no longer work on it. Only click OK if that is what you want. Otherwise, press Cancel.');">Click here to finish your exam.</button>
```

Comment:
To make this look nicer, could we do it with the existing confirmFinishExam modal? This can be triggered like this:
```
<button class="btn btn-danger" data-toggle="modal" data-target="#confirmFinishModal">Finish assessment</button>
```

The modal itself is still in the code (although unused) at:

https://github.com/PrairieLearn/PrairieLearn/blob/master/pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs#L13